CentralityiOS/CentralityiOS/ViewControllers/ToDoFeedViewController.m

```diff
@@ -84,9 +84,19 @@ - (void)fetchData{
         }
         [self.taskTableView reloadData];
         [self.refreshControl endRefreshing];
+        [self detectEmptyFeed];
     }];
 }
 
+- (void)detectEmptyFeed{
+    if ([self arrayOfTasks].count == 0){
+        self.feedMessageLabel.hidden = FALSE;
+    }
+    else{
+        self.feedMessageLabel.hidden = true;
+    }
+}
+
```

Comment:
Is this equivalent to `self.feedMessageLabel.hidden = ![self arrayOfTasks].count == 0`?

Reply:
Wow it is, I'll update that right now.